spec/metanorma_spec.rb

```diff
@@ -140,32 +140,34 @@ def ruby(x)
   it "non-zero exit code when metanorma compile for missing file" do
     expect(system("metanorma compile not_existing.adoc")).to be false
   end
-end
 
-RSpec.describe "warns when no standard type provided" do
-  file "test.adoc", ASCIIDOC_CONFIGURED_HDR
-  command "metanorma test.adoc"
-  its(:stdout) { is_expected.to include "Please specify a standard type" }
-end
+  it "warns when no standard type provided" do
+    File.write("test.adoc", ASCIIDOC_CONFIGURED_HDR, encoding: "UTF-8")
+    stdout = `metanorma test.adoc`
```

Comment:
Good idea, if you need more control over the `stdout` or `stderr` then I think we also have test helpers here - https://github.com/metanorma/metanorma-cli/blob/master/spec/support/console_helper.rb

Reply:
`` or `Open3`, looks much cleaner and native, what benefits of `capture_stdout`/`capture_stderr` against them?

Reply:
I'm not exactly sure anymore why we wrote our own version, but I think there was some issue with RSpec or something back then. If there is no incompatibility with `open3` and our test suite then please feel free to migrate it 👍